src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaSelection/tests/MediaSelection.test.js

```diff
@@ -0,0 +1,294 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import {mount, render, shallow} from 'enzyme';
+import pretty from 'pretty';
+import React from 'react';
+import datagridAdapterRegistry from 'sulu-admin-bundle/containers/Datagrid/registries/DatagridAdapterRegistry';
+import MediaSelection from '../MediaSelection';
+import MediaCardSelectionAdapter from '../../Datagrid/adapters/MediaCardSelectionAdapter';
+import MediaSelectionStore from '../stores/MediaSelectionStore';
+
+jest.mock('../stores/MediaSelectionStore', () => jest.fn());
+
+jest.mock('sulu-admin-bundle/containers', () => {
+    return {
+        Datagrid: require('sulu-admin-bundle/containers/Datagrid/Datagrid').default,
```

Comment:
Was there now any reason not to import that with the `import` above?

Reply:
The first comment was referring to the 'datagridAdapterRegistry' which is not included through `import`. I have to load the `Datagrid` this way because it is loaded through `named imports` inside the code in the `sulu-media-bundle`